data/ships.txt

```diff
@@ -1522,7 +1522,7 @@ ship "Fury"
 		"nGVF-BB Fuel Cell"
 		"LP036a Battery Pack"
 		"D14-RN Shield Generator"
-		"Small Radar Jammer" 2
```

Comment:
This would have the effect of increasing the Fury's price by 110,000 credits, which seems a bit steep for an interceptor--even a heavy one. There may be room for a large jammer now, but I'm not sure the price increase could justify it.

Reply:
Good point. The Hawk also has two small radar jammers, so I suppose that's more than enough for the Fury. Changed.